Description: Define unknown host CPU types as CPU_unknown
 The latest upstream version of fs-uae introduced new JIT
 support for x86-based architectures. Unfortunately, this
 change also made the build bail out if the host CPU is not
 among the list of supported types. With this patch, unknown
 host CPUs no longer trigger an error but will just define
 the CPU type as 'CPU_unknown'.
 .

--- fs-uae-2.8.0+dfsg.orig/src/include/sysdeps.h
+++ fs-uae-2.8.0+dfsg/src/include/sysdeps.h
@@ -50,7 +50,7 @@ using namespace std;
 #elif defined(__powerpc__) || defined(__ppc__) || defined(_M_PPC)
 #define CPU_powerpc 1
 #else
-#error unrecognized CPU type
+#define CPU_unknown 1
 #endif
 
 #ifdef _WIN32
